addons/hw_drivers/tools/helpers.py

```diff
@@ -61,14 +61,33 @@ def run(self):
 elif platform.system() == 'Linux':
     @contextlib.contextmanager
     def writable():
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/"])
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"])
         try:
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/"], check=True)
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"], check=True)
             yield
         finally:
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"])
+            try:
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,ro", "/"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
+            except subprocess.CalledProcessError:
+                _logger.error("Failed to remount read-only")
```

Comment:
is there a reason to hide the actual error itself ? May be better to use `_logger.exception` as it would contains the stackpath